### DISCUSSION TITLE
Disable / Hide Page Components

### DISCUSSION THREAD

**Discussion by rafzielinski** [flyntwp/flynt#363] [Q&A] [answered]
Hi, first things first, amazing theme!

I'm wondering if it's possible to hide or disable Page Components area when adding Custom Post Type.
I know it can be done in the Screen Options drawer, but I think that will affect just the current user account and I'd prefer to set it globally for Post Type.

**Comment by steffenbew** [comment, marked as answer]
Hi @rafzielinski, the page components can be defined to show up on specific post types only. This can be achieved by adjusting the conditional in the ACF field group. By default, the page components are defined to show up on any post type that is not the default post type: https://github.com/flyntwp/flynt/blob/master/inc/fieldGroups/pageComponents.php#L30-L38

**Reply by rafzielinski** [reply]
Hi @steffenbew, thanks for the response, I totally missed that :facepalm: although, I've found a workaround to hide it from view via CSS.
